### PR TITLE
[Bugfix] Fix rotations tomo X

### PR DESCRIPTION
**Pull request by chMoussa**
Solves #45

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ classifiers=[
 ]
 dependencies = [
     "torch",
-    "qadence[braket, pulser, horqrux]",
+    "qadence[braket, pulser, horqrux]<1.8.0",
```
Comment (RolandMacDoland) [inline pyproject.toml]: I guess you can remove the braket dependency ? I wouldn't set an upper bound for the versions though.

Comment (chMoussa) [reply pyproject.toml]: Ok, let me fix it to 1.7.8 and remove pulser then

File: pyproject.toml [hunk 2]
```diff
@@ -30,7 +30,7 @@ classifiers=[
 ]
 dependencies = [
     "torch",
-    "qadence[braket, pulser, horqrux]",
+    "qadence[pulser, horqrux]==1.7.8",
```
Comment (RolandMacDoland) [inline pyproject.toml:33]: Still unsure about pinning the versions.

**Review by RolandMacDoland — APPROVED** [review]
LGTM ! Thanks.

File: pyproject.toml (hunk 2 above)
Comment (chMoussa) [reply pyproject.toml:33]: I'll fix it in another MR as we have changed to using Noise handler instead of Noise in qadence

**Pull request merged**